art/art.py

```diff
@@ -139,19 +139,23 @@ def aprint(artname, number=1):
         if artname == "UnicodeEncodeError":
             raise UnicodeEncodeError(
                 'test', u"", 42, 43, 'test unicode-encode-error')
-        print(art(artname=artname, number=number))
+        result, artname = art(
+            artname=artname, number=number, __detailed_return=True)
+        print(result)
     except UnicodeEncodeError:
         print(ART_ENVIRONMENT_WARNING.format(artname))
```

Comment:
@sadrasabouri I think we should update `ART_ENVIRONMENT_WARNING` to something like this:
`"[Warning] '{0}' art is not printable in this environment."`